src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -94,45 +94,59 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
                 _logger.ScriptStartNotLoadingExtensionBundle("WARMUP_LOG_ONLY", bundleConfigured, isPrecompiledFunctionApp, isLegacyExtensionBundle);
             }
 
+            string baseProbingPath = null;
+
             if (bundleConfigured && (!isPrecompiledFunctionApp || _extensionBundleManager.IsLegacyExtensionBundle()))
             {
-                extensionsPath = await _extensionBundleManager.GetExtensionBundleBinPathAsync();
-                if (string.IsNullOrEmpty(extensionsPath))
+                extensionsMedatadaPath = await _extensionBundleManager.GetExtensionBundleBinPathAsync();
```

Comment:
I think this was a typo?
```suggestion
                extensionsMetadataPath = await _extensionBundleManager.GetExtensionBundleBinPathAsync();
```

Reply:
Yes, it was as a result of the last changes based on feedback. Made the changes throughout as the variable is used in multiple locations.